README.md

```diff
@@ -1,5 +1,12 @@
 # Guess The Number
+## Fork coded by: Jena Durell
 
+ - Imported functions from ask.js file in order to help with refactoring, see https://www.stanleyulili.com/node/node-modules-learn-how-to-import-and-use-functions-from-another-file/
+
+ - Attempted to use imported functions for each separate game as well, but encountered circular dependency errors despite multiple attempts to fix them. 
+
+ - Reverted back to single file (index.js) for the game, able to successfully import ask.js into game.
+---
```

Comment:
Just want to say that these are great notes, and a great addition to show your coding process :)

Reply:
Thank you! I like updating the README and looking back and thinking about everything I did and tried. I think it's also going to come in handy as we get deeper into the class- citing the article here is a good way for me to keep track of it too!